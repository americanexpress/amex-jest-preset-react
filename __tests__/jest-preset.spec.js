test('a jest configuration is exported', () => {
  const jestPreset = require('../jest-preset');

  expect(jestPreset).toMatchSnapshot({
    setupFilesAfterEnv: expect.any(Array),
    testResultsProcessor: expect.any(String),
    cache: expect.any(Boolean)
  });
});