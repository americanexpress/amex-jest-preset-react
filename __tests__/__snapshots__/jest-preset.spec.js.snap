// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a jest configuration is exported 1`] = `
Object {
  "cache": Any<Boolean>,
  "cacheDirectory": "<rootDir>/.jest-cache",
  "collectCoverage": true,
  "collectCoverageFrom": Array [
    "src/**/*.{js,jsx}",
    "!**/node_modules/**",
    "!test-results/**",
  ],
  "coverageDirectory": "<rootDir>/test-results/coverage",
  "coverageReporters": Array [
    "text",
    "lcov",
    "cobertura",
  ],
  "coverageThreshold": Object {
    "global": Object {
      "branches": 100,
      "functions": 100,
      "lines": 100,
      "statements": 100,
    },
  },
  "moduleNameMapper": Object {
    "\\\\.(css|scss)$": "identity-obj-proxy",
  },
  "modulePathIgnorePatterns": Array [
    "npm-cache",
    ".npm",
  ],
  "setupFilesAfterEnv": Any<Array>,
  "snapshotSerializers": Array [
    "enzyme-to-json/serializer",
  ],
  "testEnvironment": "jsdom",
  "testResultsProcessor": Any<String>,
}
`;
